model/GameModel.ts

```diff
@@ -22,18 +103,91 @@ export class GameModel {
         return this._imageUrl;
     }
 
-    get birthYear(): number {
-        return this._birthYear;
+    get birth(): Hint<Date> {
```

Comment:
All of this is boilerplate, and could be retrieved using our `HintList` interface with `hintList.death`, `hintList.occupation`, etc. without a need for that many getters.

Reply:
I'll look into it, the HintList interface wouldn't be filterable which is an issue but let me think about how I could implement that :)

Reply:
Thanks ! Think about making the HintList a child of official Lists, so as to override methods we want to use such as `filter` for instance.

Reply:
actually having a HintList interface is not possible as, interfaces in typescript cannot have functions or methods. 
We could have a HintList class.

Reply:
You might want to have a look to my latest commit, I decided to make HintList a class for reasons explained in the commit message, moreover it was way easier to implement a "toList()" function on which we will call filter when needed then write a filter function which would take a predicate as parameter.